testing/mocha_example.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Run this example with:
+ *
+ * deno run --location=http://local ./testing/mocha_example.ts
+ */
+import mocha from "https://cdn.skypack.dev/mocha@9.1.2?dts";
+import chai from "https://cdn.skypack.dev/chai@4.3.4?dts";
+
+function add(a: number, b: number): number {
+  return a + b;
+}
+
+describe("add", () => {
+  it("should add two positive numbers correctly", () => {
+    chai.expect(add(2, 3)).to.equal(2);
+  });
+});
```

Comment:
This may work for a single file, but does it work if tests are scattered across many files? I don't see this being super useable just because someone would have to import all these files somehow. Maybe it would work with a dynamic import that walks the directory tree?

I think the mocha pollyfill Luca has (currently in a gist) would be better to clean up and then recommend: https://github.com/denoland/deno/pull/12190#issuecomment-933441089 -- I guess the problem though is that the polyfill is not 100%.

Reply:
I agree with David. @cmorten could you remove the mocha example for now? I think it's still valuable to land chai and sinon examples as well as add them to the Manual.